bridge/client/app/app-header/app-header.component.ts

```diff
@@ -54,46 +54,33 @@ export class AppHeaderComponent implements OnInit, OnDestroy {
         if(keptnInfo.versionCheckEnabled === null) {
           this.showVersionCheckInfoDialog();
         } else if(keptnInfo.versionCheckEnabled) {
-          keptnInfo.keptnVersionInvalid = !this.doVersionCheck(keptnInfo.keptnVersion, keptnInfo.availableVersions.cli.stable, keptnInfo.availableVersions.cli.prerelease, "Keptn");
-          keptnInfo.bridgeVersionInvalid = !this.doVersionCheck(keptnInfo.bridgeInfo.bridgeVersion, keptnInfo.availableVersions.bridge.stable, keptnInfo.availableVersions.bridge.prerelease, "Keptn Bridge");;
+          keptnInfo.keptnVersionInvalid = !this.doVersionCheck(keptnInfo.keptnVersion, keptnInfo.availableVersions.cli.stable);
```

Comment:
As discussed with @johannes-b again, we will keep the functionality to notify about CLI and Bridge version updates separately. Therefore please allow following cases:
1. New CLI x.y.z available
2. New Bridge x.y.z available
3. New CLI x.y.z and Bridge x.y.z available